IMPORTING-DATA.md

```diff
@@ -0,0 +1,311 @@
+people to enter their postcode and be pointed to their local services.
+
+It is composed of broadly 2 things:
+
+1.  A database of postcodes and the location of their geographical
+    centre
+2.  A database of administrative entities (e.g. local councils) and
+    their boundaries on a map
+
+When a user enters a postcode on GOV.UK, MapIt will look it up to find
+the geographical centre and then return information about the
+administrative entities that postcode belongs to. We then use this
+information to refer the user to local services provided by those
+entities.
+
+## Error Alerting
+
+Errors from the web application are emailed to the `BUG_EMAIL` addresses in [alphagov-deployment](https://github.digital.cabinet-office.gov.uk/gds/alphagov-deployment/blob/master/mapit/settings)
+
+You can access these via Google groups.
+
+## Updating Datasets
+
+Datasets for boundary lines are published twice a year (in May and
+October) and postcodes are released four times a year (in February, May,
+August and November). This means over time our database will become more
+and more out of date and users will start to complain. When they do (or
+before if possible) we should update the data.
+
+To update a live mapit server we:
+
+1.  [Generate a new database](#generate-a-new-database)
+2.  [Export new database to S3](#export-new-database-to-s3)
+3.  [Update servers with new database](#update-servers-with-new-database)
+
+### Generate a new database
+
+1.  Checkout the [Mapit](https://github.com/alphagov/mapit) repo to your
+    dev VM if you don't have it already.
+2.  Prepare your mapit repo so that you can run the importer scripts:
+    a)  Run `startup.sh` - this will install all dependencies and run
+        the server, once the server is running you can kill it, we just
+        wanted the dependencies installed.
+    b)  Prepare your database for importing new data by running the
+        following to create an empty database and migrate it to the
+        empty mapit schema:
+
+            $ sudo -u postgres dropdb mapit
+            $ sudo -u postgres createdb --owner mapit mapit
+            $ sudo -u postgres psql -c "CREATE EXTENSION postgis; CREATE EXTENSION postgis_topology;" mapit
+            $ govuk_setenv mapit .venv/bin/python ./manage.py migrate
+
+3.  Checkout the [mapit-scripts](https://github.digital.cabinet-office.gov.uk/gds/mapit-scripts)
+    repo to your dev VM if you don't have it already.
+
+4.  Find the latest ONS Postcode Database, Boundary Line, and
+    OSNI datasets.
+    a)  MySociety may have mirrored the latest datasets on their cache
+        server: <http://parlvid.mysociety.org/os/> so check there first.
+    b)  ONSPD releases can be found via the Office for National
+        Statistics (ONS) at
+        <http://www.ons.gov.uk/ons/guide-method/geography/products/postcode-directories/-nspp-/index.html>
+        and
+        <https://geoportal.statistics.gov.uk/geoportal/catalog/content/filelist.page>?&pos=7&cat=\#PP
```

Comment:
I'll admit that I mainly looked at the ones that were Github Enterprise related 🙈